### DISCUSSION TITLE
How to hide gear wheel button?

### DISCUSSION THREAD

**Discussion by C1eriC** [valinet/ExplorerPatcher#222] [Q&A] [answered]
How to hide gear wheel button? [This one](https://hsto.org/webt/zj/e1/vx/zje1vxipqyptt3ue0jh35it8qhi.png).

**Comment by Gaurav-Original-ClassicShellTester** [comment, marked as answer]
There's an option "Hide Control Center button" in Explorer Patcher Settings/Properties -> System tray. After that you will have to log off and logon again OR terminate Explorer.exe using Task Manager and start it again (or end sihost.exe so Explorer restarts).

**Reply by C1eriC** [reply]
It's works. thx. But I switched to another monitor and, after that, this button appears again. Can I hide it on every monitors?

**Reply by valinet** [reply]
It’s only on the main taskbar. If you play around with that, sometimes that happens. Also, if it doesn’t stay hidden, idk what else to say or do, then it just doesn’t want to respect the setting and that’s mostly it. I haven’t experienced that, but indeed, I don’t undock my laptop much these days. Anyway, it’s just a tiny button…

**Reply by p-bogi** [reply]
Thanks! I installed ExplorerPatcher yesterday and this icon was annoying in left/right taskbar position I'm using - because it takes the whole taskbar width!
Better to use that place for example for few small toolbar icons to easily launch frequently used apps. I like toolbar icons because you can drop a file on them.

**Reply by ghost** [reply]
To Get Rid of Gear Wheel in the lastest version of ExplorerPatcher:

1. Press Win+X and choose "Properties".

2. Go to Enable Missing System Tray Icons, set to this default icons and this gets rid of Gear Wheel.

3. Pick System Tray Icons you desire.

DONE

**Comment by C1eriC** [comment]
I found out, that this is gear button is appeared after pressing Win+A. If restart explorer.exe it disappear again.

**Reply by C1eriC** [reply]
And, after that ↑ button stop work, nothing happens if I press to it. And hotkey Win+A don't work too.
